### ISSUE TITLE
Fix errors

### ISSUE THREAD

**Issue by steve-hb** [E-Edu/frontend#187]
fix the errors showed in sentry and the console

**Comment by steve-hb** [comment]
Sorry for the missing description!

In our Sentry system are many errors logged. These errors have to be evaluated and fixed since it's logging every few hours.

Examples:

```
i18next::translator: missingKey en translation component.subject.label.unkown Unknown
--
```

```
Warning: Each child in a list should have a unique "key" prop. See https://fb.me/react-warning-keys for more information.%s      in MenuElement (at Sidebar.js:143)     in div (at Sidebar.js:142)     in div (at Sidebar.js:140)     in Sidebar (at PageLayout.js:32)     in div (at PageLayout.js:30)     in PageLayout (created by Context.Consumer)     in withRouter(PageLayout) (at App.js:25)     in Router (created by BrowserRouter)     in BrowserRouter (at App.js:24)     in App (at src/index.js:13)
--
```

```
Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in %s.%s the componentWillUnmount method      in dashboard (created by Context.Consumer)
--
```

```
Cannot read property 'value' of null
Profile: 19
        this.state.username = document.getElementById('usernameInput').value;
```

**Comment by steve-hb** [comment]
@TimyStream @Stefan080106 I think u both have Sentry accounts, don'
t u?

**Comment by TimyStream** [comment]
No, i dont have one at the moment 😊

 

Von: Steve <notifications@github.com> 
Gesendet: Sonntag, 5. April 2020 17:56
An: E-Edu/frontend <frontend@noreply.github.com>
Cc: TimyStream <timy@timystream.de>; Mention <mention@noreply.github.com>
Betreff: Re: [E-Edu/frontend] Fix errors (#187)

 

@TimyStream <https://github.com/TimyStream>  @Stefan080106 <https://github.com/Stefan080106>  I think u both have Sentry accounts, don't u?
